tests/conftest.py

```
@@ -7817,20 +7833,112 @@ def _teardown():
+                            f"/local-volume/mounts/{pv_name}"
+                        )
+                        run_cmd(
+                            f"oc debug node/{node_name} -- chroot /host"
+                            f" bash -c '"
+                            f"umount {mount_path} 2>/dev/null;"
+                            f" rm -rf {mount_path};"
+                            f" wipefs -a {disk_path};"
+                            f" mkfs.ext4 -F {disk_path}'",
+                            timeout=300,
+                        )
+                    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)